ansible/roles/manage-server-prod/templates/application.yml.j2

```diff
@@ -17,17 +17,26 @@ server:
 features: {{ manage.features }}
 
 push:
-  url: https://{{ engine_api_domain }}/api/connections
-  user: {{ engine_api_metadata_push_user }}
-  password: {{ engine_api_metadata_push_password }}
-  name: {{ instance_name }} EngineBlock
-  exclude_edugain_imports: {{ manage_exclude_edugain_imports_in_push }}
+  eb:
+    url: https://{{ engine_api_domain }}/api/connections
+    user: {{ engine_api_metadata_push_user }}
+    password: {{ engine_api_metadata_push_password }}
+    name: {{ instance_name }} EngineBlock
+    exclude_edugain_imports: {{ manage_exclude_edugain_imports_in_push }}
+    exclude_oidc_rp: {{ manage_exclude_oidc_rp_imports_in_push }}
+  oidc:
+    url: https://oidcng.{{ base_domain }}/manage/connections
+    user: manage
+    name: {{ manage.oidcng_name }}
+    password: {{ oidcng_api_metadata_push_password }}
+    enabled: {{ manage.oidc_push_enabled }}
```

Comment:
By default the `manage.oidc_push_enabled` is set to be "False", in order to prevent us from having to update this value it would be preferred to have it be True by default.